easybuild/tools/github.py

```diff
@@ -969,9 +969,37 @@ def not_eligible(msg):
     else:
         res = not_eligible(msg_tmpl % 'no milestone found')
 
+    # check whether all mentioned easyconfig PR dependencies have been merged
+    msg_tmpl = "* PR dependencies: %s"
+    if not pr_data.get('unmerged_pr_deps', False):
+        print_msg(msg_tmpl % 'OK (no unmerged PR dependencies found)', prefix=False)
+    else:
+        unmerged_pr_deps = ', '.join(pr_data['unmerged_pr_deps'])
+        res = not_eligible(msg_tmpl % 'FAILED (%s not yet merged)' % unmerged_pr_deps)
```

Comment:
@migueldiascosta I would reverse the logic here, seems easier to understand. Also, we should always include `'unmerged_pr_deps'` in `pr_data` (see also below)?

```python
if pr_data['unmerged_pr_deps']:
    unmerged_pr_deps = ', '.join(pr_data['unmerged_pr_deps'])
    res = not_eligible(msg_tmpl % 'FAILED (%s not yet merged)' % unmerged_pr_deps)
else:
    print_msg(msg_tmpl % 'OK (no unmerged PR dependencies found)', prefix=False)